Cargo install nu is broken

### Describe the bug

cargo install nu
    .... 
   Compiling procfs-core v0.17.0
   Compiling chrono-humanize v0.2.3
   Compiling parse_datetime v0.6.0
   Compiling dtparse v2.0.1
   Compiling calamine v0.26.1
error[E0107]: struct takes 1 generic argument but 0 generic arguments were supplied
   --> /home/username/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/calamine-0.26.1/src/xlsb/mod.rs:603:18
    |
603 |     r: BufReader<ZipFile<'a>>,
    |                  ^^^^^^^ expected 1 generic argument
    |
note: struct defined here, with 1 generic parameter: `R`
   --> /home/username/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/zip-2.6.1/src/read.rs:252:12
    |
252 | pub struct ZipFile<'a, R: Read> {
    |            ^^^^^^^     -
help: add missing generic argument
    |
603 |     r: BufReader<ZipFile<'a, R>>,
    |                            +++

error[E0107]: struct takes 1 generic argument but 0 generic arguments were supplied
   --> /home/username/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/calamine-0.26.1/src/xlsx/mod.rs:26:52
    |
26  | pub(crate) type XlReader<'a> = XmlReader<BufReader<ZipFile<'a>>>;
    |                                                    ^^^^^^^ expected 1 generic argument
    |
note: struct defined here, with 1 generic parameter: `R`
   --> /home/username/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/zip-2.6.1/src/read.rs:252:12
    |
252 | pub struct ZipFile<'a, R: Read> {
    |            ^^^^^^^     -
help: add missing generic argument
    |
26  | pub(crate) type XlReader<'a> = XmlReader<BufReader<ZipFile<'a, R>>>;
    |                                                              +++

error[E0107]: struct takes 1 generic argument but 0 generic arguments were supplied
   --> /home/username/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/calamine-0.26.1/src/ods.rs:24:42
    |
24  | type OdsReader<'a> = XmlReader<BufReader<ZipFile<'a>>>;
    |                                          ^^^^^^^ expected 1 generic argument
    |
note: struct defined here, with 1 generic parameter: `R`
   --> /home/username/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/zip-2.6.1/src/read.rs:252:12
    |
252 | pub struct ZipFile<'a, R: Read> {
    |            ^^^^^^^     -
help: add missing generic argument
    |
24  | type OdsReader<'a> = XmlReader<BufReader<ZipFile<'a, R>>>;
    |                                                    +++

   Compiling terminal_size v0.4.2
   Compiling xattr v1.5.0
   Compiling miette v7.5.0
   Compiling clap_builder v4.5.36
   Compiling which v7.0.3
   Compiling fd-lock v4.0.4
   Compiling tempfile v3.19.1
   Compiling toml v0.8.20
   Compiling trash v5.2.2
   Compiling human-date-parser v0.2.0
   Compiling roxmltree v0.20.0
   Compiling termcolor v1.4.1
   Compiling filesize v0.2.0
   Compiling nu v0.103.0
   Compiling alphanumeric-sort v1.5.3
   Compiling simplelog v0.12.2
For more information about this error, try `rustc --explain E0107`.
error: could not compile `calamine` (lib) due to 3 previous errors
warning: build failed, waiting for other jobs to finish...
error: failed to compile `nu v0.103.0`, intermediate artifacts can be found at `/tmp/cargo-installudVTcA`.
To reuse those artifacts with a future compilation, set the environment variable `CARGO_TARGET_DIR` to that path.


### How to reproduce

1. type "cargo install nu"

### Expected behavior

I expected nu to install

### Configuration

cargo 1.86.0

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -71,7 +71,7 @@ brotli = "7.0"
 byteorder = "1.5"
 bytes = "1"
 bytesize = "1.3.3"
-calamine = "0.27"
+calamine = "0.26"
 chardetng = "0.1.17"
 chrono = { default-features = false, version = "0.4.34" }
 chrono-humanize = "0.2.3"

